src/k8s/pkg/k8sd/types/cluster_config.go

```diff
@@ -112,4 +172,93 @@ func ClusterConfigFromBootstrapConfig(b *apiv1.BootstrapConfig) ClusterConfig {
 			Port: b.K8sDqlitePort,
 		},
 	}
+
+	for _, component := range b.Components {
+		switch component {
+		case "network":
+			config.Network.Enabled = &[]bool{true}[0]
```

Comment:
ok, here you cannot use `new(bool)` but please just create a variable and reference it. Creating 7 slices to get bool references sounds unnecessary.

Reply:
or better just create a `utils.Pointer(true)` as Angelos suggested